xml/System.Xml/XmlException.xml

```diff
@@ -347,9 +347,39 @@ The `XmlException` is a generic XML-related error that is thrown if problems occ
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the line number indicating where the error occurred.</summary>
-        <value>The line number indicating where the error occurred.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the 1-based line number indicating where the error occurred.</summary>
+        <value>The 1-based line number indicating where the error occurred.</value>
+        <remarks>
```

Comment:
Looking at this more closely I don't think you need to show the example at all (unless the example was specific about using the property). I think I'd just add the info at the summary.

Reply:
I actually think it makes sense to have a sample since a lot of people don't know how to set line info

Reply:
But the example here doesn't use the API at all. It just proves that the property is 1-based.

Reply:
Maira has a good point. An example like this should be located in the `Load` API.

Instead of a full code example, I will simply mention that if we attempt to load a string that looks like `"<<foo>\r\nbar</foo>"`, then we will throw an exception with the specified message. Does that sound better, @mairaw?

@krwq can you please add an example (with a valid xml string) to the Load method, if there isn't one yet?